framer/Screen.coffee

```diff
@@ -24,6 +24,15 @@ class ScreenClass extends BaseClass
 	toInspect: ->
 		return "<Screen #{Utils.roundWhole(@width)}x#{Utils.roundWhole(@height)}>"
 
+	# Point Conversion
+
+	convertPointToLayer: (point, layer) =>
+		return Utils.convertPointFromContext(point, layer, false, true)
+
+	convertPointToCanvas: (point) =>
+		ctx = Framer.CurrentContext.device.context
```

Comment:
OK, just wanted to confirm. I get nervous when I see code that tries to access deeply-nested properties :)